app/ui-react/packages/ui/src/Shared/HelpDropdown.tsx

```diff
@@ -13,6 +13,11 @@ export interface IHelpDropdownProps {
   className?: string;
   isOpen: boolean;
   launchAboutModal: any;
+  launchSupportPage: any;
+  launchSampleIntegrationTutorials: any;
+  launchUserGuide: any;
+  launchConnectorsGuide: any;
+  launchContactUs: any;
```

Comment:
So you decided to keep the `any` type?

Reply:
Nope, just missed these!

Reply:
Just realized I hadn't pressed the save button after making the changes the first time 🤦‍♂️